Formula/buildkit.rb

```diff
@@ -0,0 +1,21 @@
+class Buildkit < Formula
+  desc "Сoncurrent, cache-efficient, and Dockerfile-agnostic builder toolkit"
+  homepage "https://github.com/moby/buildkit"
+  url "https://github.com/moby/buildkit/archive/v0.3.3.tar.gz"
+  sha256 "07a44de2bcbaa24055d1a7c9223be9483eb5ec68bf79c9e51c7d583a9b430820"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/moby/buildkit/").install Dir["*"]
+
+    ldflags = ["-X github.com/moby/buildkit/version.Version=#{version}",
+               "-X github.com/moby/buildkit/version.Package=github.com/moby/buildkit"]
+    system "go", "build", "-o", bin/"buildctl", "-ldflags", ldflags.join(" "), "github.com/moby/buildkit/cmd/buildctl"
```

Comment:
No man page, no documentation?

Reply:
Added `README.md` and files from `docs/*.md`.